doc/man1/openssl-verification-options.pod

```diff
@@ -426,6 +532,133 @@ This option has no effect and is retained for backward compatibility only.
 
 =back
 
+=head2 Extended Key Usage
+
+A certificate may contain Extended Key Usage (EKU) extensions,
+which restrict the purposes the certificate can be used for.
+The checks for such restrictions are rather complex and include various
+hacks and workarounds to handle broken certificates and software.
+
+The same code is used when verifying untrusted certificates in chains
+so this section is useful if a chain is rejected by the verify code.
```

Comment:
I find this sentence pretty weird, too, but I had just moved the whole section including it from `doc/man1/openssl-x509.pod.in` to here.

I've added "and intermediate CA certificates" to the paragraph before, which now reads
```
Options like B<-purpose> lead to checking the certificate extensions,
which determine what the target certificate and intermediate CA certificates
can be used for.
The checks for such restrictions are rather complex and include various
hacks and workarounds to handle broken certificates and software.
```
and simply dropped the strange sentence because it does not add much enlightenment.

BTW, I just noticed that 
`=head2 Extended Key Usage`
was wrongly placed and other (sub-)headers were missing; I've corrected this now.